ethcore/light/src/provider.rs

```diff
@@ -33,6 +33,9 @@ use transaction_queue::TransactionQueue;
 
 use request;
 
+/// Maximum allowed size of a headers request.
+pub const MAX_HEADERS_LENGTH: u64 = 512;
```

Comment:
I personally find the previous name (`MAX_HEADERS_TO_SEND`) more descriptive and less confusing than this one.

Reply:
The reason I changed it is that the variable was used in a context that was receiving headers, not sending them, so it seemed confusing. What about `MAX_HEADERS_PER_REQUEST`?

Reply:
I like `MAX_HEADERS_PER_REQUEST` much better than `MAX_HEADERS_LENGTH`.